apps/daimo-mobile/src/view/shared/error.tsx

```diff
@@ -16,3 +22,86 @@ export function ErrorRowCentered({ error }: { error: { message?: string } }) {
     </TextCenter>
   );
 }
+
+export function ErrorBanner({
+  error,
+  displayTitle,
+  displayMessage,
+  showDownloadButton,
+}: {
+  error?: { message?: string };
+  displayTitle: string;
+  displayMessage?: string;
+  showDownloadButton?: boolean;
+}) {
+  const nav = useNav();
+  const goHomeScreen = () => nav.popToTop();
+  const goAppStore = () => {};
+  console.log(
+    `[ERROR] rendering ${error?.message || "Unknown message"}`,
+    error
+  );
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.closeCircle}>
+        <Image source={Close} style={styles.closeIcon} />
```

Comment:
recommend using the same octicon we use for close elsewhere--see ScreenHeader for example

you can still make the color red if there's a good reason to do so. otherwise, i'd default to same color ^

Reply:
I used png from figma because it looked ever so slightly different but I gotcha that in terms of code clarity using the octicon component makes so much more sense 🫡